File: 백준/Bronze/1546. 평균/평균.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
const score = input[1].split(' ');
const max =  Math.max(...score);

let result = 0;
for(let i = 0; i < input[0]; i++){
  result += score[i] / max * 100;
}

console.log(result / input[0]);

File: 백준/Bronze/1546. 평균/평균.cc
#include <iostream>

using namespace std;

int main() {
    int N = 0;
    int max = 0;
    int scoreArr[1000];

    cin >> N;

    for(int i = 0; i < N; ++i){
        cin >> scoreArr[i];

        if(max < scoreArr[i]){
            max = scoreArr[i];
        }
    }

    double sum = 0;
    for(int i = 0; i < N; ++i){
        sum += ((double)scoreArr[i] / max) * 100;
    }

    double result;
    result = sum / N;
    
    cout << result;

    return 0;
}